dask/dask.sh

```diff
@@ -35,7 +35,7 @@ readonly MASTER="$(/usr/share/google/get_metadata_value attributes/dataproc-mast
 readonly DASK_LAUNCHER=/usr/local/bin/dask-launcher.sh
 readonly DASK_SERVICE=dask-cluster
 
-CONDA_PACKAGES=("dask")
+CONDA_PACKAGES=("dask>=2.26")
```

Comment:
Seems like we can extract `dask/dask.sh` changes to a separate PR?

Reply:
Done